src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_Routine_DropAndCreateRelease.sql

```diff
@@ -0,0 +1,9 @@
+CREATE PROCEDURE DropAndCreateRelease @Release dbo.ReleaseType READONLY
```

Comment:
This file isn't run

Reply:
Hah well spotted sir - teach me to add things to the db manually!

Reply:
OK fxied